addons/goldpinger/template/values-bloomberg.yaml

```diff
@@ -0,0 +1,76 @@
+# Values for Bloomberg goldpinger chart
+# Based on https://github.com/bloomberg/goldpinger/tree/main/helm-chart
+
+image:
+  repository: bloomberg/goldpinger
+  tag: "3.10.2"
+  pullPolicy: IfNotPresent
+
+# Resource limits aligned with existing kURL values
+resources:
+  limits:
+    cpu: 50m
+    memory: 128Mi
+  requests:
+    cpu: 20m
+    memory: 64Mi
+
+# Service account configuration
+serviceAccount:
+  create: true
+  name: goldpinger
+
+# Service configuration to match existing kURL setup
+service:
+  type: ClusterIP
+  port: 80
+
+# Pod configuration with environment variables matching existing setup
+extraEnv:
+  # podIP is used to select a randomized subset of nodes to ping.
+  - name: POD_IP
+    valueFrom:
+      fieldRef:
+        fieldPath: status.podIP
+  - name: PING_TIMEOUT
+    value: "2s"
+  - name: CHECK_TIMEOUT
+    value: "3s"
+  - name: CHECK_ALL_TIMEOUT
+    value: "10s"
+
+# RBAC configuration
+rbac:
+  create: true
+
+# Pod settings
+goldpinger:
+  port: 8080
```

Comment:
a lot in this file is the defaults. i feel like it is better to have a minimal values file with only what is necessary for better forwards compatibility